lang/ilias_es.lang

```diff
@@ -2729,7 +2728,7 @@ chatroom_adm#:#client_settings#:#Configuración del cliente
 chatroom_adm#:#enable_osd#:#Activar notificaciones en pantalla
 chatroom_adm#:#general_settings_title#:#Configuraciones generales del chat
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Intervalo de refresco
+chatroom_adm#:#osd_interval#:#Intervalo de refresco
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.